src/components/sign-in/sign-in.component.jsx

```diff
@@ -15,10 +15,18 @@ class SignIn extends React.Component{
         }
 
 }
-handleSubmit=  event =>{
+handleSubmit= async event =>{
     event.preventDefault();
-
-    this.state({email:'', password:''});
+    const {email,password} =this.state;
+
+    try{
+        await auth.signInWithEmailAndPassword(email, password);
+        this.setState({email:'', password:''});
```

Comment:
why set empty strings here ?

Reply:
just test so after login just clear the feild . After completion convert it to link to main page.